strapi/strapi
# [Blocks editor] Fix- The first time you click on the List option or button the list is not created

### What does it do?

It fixes the problem we had when we tried to create a new list, the first time you click on the button or option it doesn't work

### Why is it needed?

To solve the issue

### How to test it?

Create a new instance with a Block Field and then click directly on the list Button, you need to see the list created (same for the list option)
